internal/gengapic/genrest.go

```diff
@@ -144,23 +143,152 @@ func (g *generator) genRESTMethod(servName string, serv *descriptor.ServiceDescr
 	if pf, err := g.pagingField(m); err != nil {
 		return err
 	} else if pf != nil {
-		// TODO(dovs)
-		return nil
+		iter, err := g.iterTypeOf(pf)
+		if err != nil {
+			return err
+		}
+
+		return g.pagingRESTCall(servName, m, pf, iter)
 	}
 
 	switch {
 	case m.GetClientStreaming():
-		// TODO(dovs)
-		return nil
+		return g.noRequestStreamRESTCall(servName, serv, m)
 	case m.GetServerStreaming():
-		// TODO(dovs)
-		return nil
+		return g.serverStreamRESTCall(servName, serv, m)
 	default:
-		// TODO(dovs)
 		return g.unaryRESTCall(servName, m)
 	}
 }
 
+func (g *generator) serverStreamRESTCall(servName string, s *descriptor.ServiceDescriptorProto, m *descriptor.MethodDescriptorProto) error {
+	// Streaming calls will most likely NEVER be supported for REST clients,
+	// but the interface signature must be preserved.
+	// Making sure not to call streaming methods on a REST client, or checking for
+	// errors in a situation with mixed gRPC and REST clients, is left to user code.
+
+	inType := g.descInfo.Type[*m.InputType]
+
+	inSpec, err := g.descInfo.ImportSpec(inType)
+	if err != nil {
+		return err
+	}
+	g.imports[inSpec] = true
+
+	servSpec, err := g.descInfo.ImportSpec(s)
+	if err != nil {
+		return err
+	}
+	g.imports[servSpec] = true
+
+	p := g.printf
+	lowcaseServName := lowcaseRestClientName(servName)
+	p("func (c *%s) %s(ctx context.Context, req *%s.%s, opts ...gax.CallOption) (%s.%s_%sClient, error) {",
+		lowcaseServName, m.GetName(), inSpec.Name, inType.GetName(), servSpec.Name, s.GetName(), m.GetName())
+	p("  var resp %s.%s_%sClient", servSpec.Name, s.GetName(), m.GetName())
+	p("  return resp, fmt.Errorf(\"%s not yet supported for REST clients\")", m.GetName())
```

Comment:
Use backticks instead of double quotes for the `p()` argument so that you can use double quotes in the generated code without escaping them.